src/olympia/reviewers/tests/test_views.py

```diff
@@ -6096,6 +6096,25 @@ def test_redirect_after_review_unlisted(self):
 
         self.assertRedirects(response, self.url)
 
+    def test_review_moderator_addon_rating_read_only(self):
```

Comment:
Lets have this just after the `test_review_moderator_addon_rating_present` test as it's testing the same functionality